### DISCUSSION TITLE
Why (TH) is GPT-4chan promoted as detailed example by the next would-be web-UI star?

### DISCUSSION THREAD

**Discussion by ai-bits** [oobabooga/text-generation-webui#1968] [General]
Still trying to wrap my head around all things - running models locally and oobabooga in particular, being hyped as THE STAR web-UI for LLMs.

After some research I'm flabbergasted that GPT-4chan is prominently promoted here as the one detailed example, where "4chan" instantly sounded fishy on a mainstream site and "taken down by HF" triggered investigation.
Turns out there is actual relations to 4chan.

Irritated
G.

**Comment by BetaDoggo** [comment]
The model is supported by the ui and it's more difficult to get safely than other models. I don't see any reason why they shouldn't provide a legitimate source for it. If you don't like it then don't use it. The model isn't super relevant now anyways.
